Dockerfile

```diff
@@ -21,13 +21,13 @@ RUN make build
 # Final IMAGE
 FROM alpine:${RUNTIME_IMAGE_TAG}
 RUN apk add --no-cache \
-            ca-certificates=20220614-r0 \
-            e2fsprogs=1.45.7-r0 \
-            xfsprogs=5.10.0-r0 \
-            xfsprogs-extra=5.10.0-r0 \
-            blkid=2.37.4-r0 \
-            e2fsprogs-extra=1.45.7-r0 \
-            cryptsetup=2.3.7-r0
+            ca-certificates=20240705-r0 \
+            e2fsprogs=1.47.0-r5 \
+            xfsprogs=6.8.0-r0 \
+            xfsprogs-extra=6.8.0-r0 \
```

Comment:
This could be very **delicate** xfs lib is used by node controller to perform disk format. However,  before upgrading we have to make sure the lib is not newer than xfs kernel module (client side).

Reply:
OK I ll not upgrade it for now